spec/openapi.yaml

```diff
@@ -6402,6 +6458,70 @@ components:
             next_token:
               type: string
               description: The pagination token of the next set of results.
+    PatchLabelMyOwnNumber:
+      type: object
+      title: PatchLabelMyOwnNumber
+      required:
+        - label
+      properties:
+        label:
+          type: string
```

Comment:
should add the constraint: label is max length = 100 characters

Reply:
updated